scss/layout.scss

```diff
@@ -0,0 +1,14 @@
+.card {
+  background-color: #FFF;
+  box-shadow: 4px 4px 10px rgba(0,0,0,0.5);
+}
+
+.container {
+  margin: 0 auto;
+  max-width: 900px;
+  width: 75%;
```

Comment:
Mmm maybe change the max-width value? If this is OK, use 100%

Reply:
The max-width was based on the width of the book-detail and the book-list in-vision prototypes.
And the width was not 100% so the cards wouldn't take all the screen width on smaller devices. But maybe it's not the best way :sweat_smile: